client/lib/sites-list/list.js

```diff
@@ -567,10 +569,10 @@ SitesList.prototype.getVisible = function() {
 SitesList.prototype.getVisibleAndNotRecentNorStarred = function() {
 	return this.get().filter( function( site ) {
 		if ( user.get().visible_site_count < 12 ) {
-			return site.visible === true && ! this.isStarred( site );
```

Comment:
Yeah, it's a quick patch, but still bothers me that the naming of the function is now inconsistent with the return value of the function. Would have personally rather have seen this function left as is, with another nearly identical for `getVisibleAndNotRecent` .